data/eos-autoupdater.conf

```diff
@@ -10,4 +10,5 @@
 #  3 - apply updates
 LastAutomaticStep=3
 IntervalDays=14
+RandomizedDelayDays=7
```

Comment:
Ah... I see. Makes sense, unless we wanted always randomise the interval always, and have a separate preference as to whether we update "right away" on boot, or wait the random interval before the first update.